app/src/main/java/com/stevesoltys/seedvault/restore/install/InstallIntentCreator.kt

```diff
@@ -14,6 +14,7 @@ internal class InstallIntentCreator(
     private val installerToPackage = mapOf(
         "org.fdroid.fdroid" to "org.fdroid.fdroid",
         "org.fdroid.fdroid.privileged" to "org.fdroid.fdroid",
+        "org.fdroid.basic" to "org.fdroid.basic",
```

Comment:
Just out of curiosity - what would happen if it wasn't on this list? I only ask for the 1:1 mapping such as this one.

Reply:
As far as I can tell, it would return a null intent (nothing to install the app with), but as far as the user experience, I'd have to look further.